src/script.js

```diff
@@ -1,3 +1,41 @@
 "use strict";
 
 // TODO - write your code here.
+
+const randomDamage = () => Math.floor(Math.random() * 10) + 1;
+
+const chooseOption = (opt1, opt2) => {
+    const randNum = Math.floor(Math.random() * 2);
+    return randNum === 0 ? opt1 : opt2;
```

Comment:
Check for understanding: why does this function use the `return` keyword but the other functions don't?